.prettierignore

```diff
@@ -1,9 +1,12 @@
-# Based on .gitignore:
-out
-dist
-node_modules
-.vscode-test/
-*.vsix
-
-# Based on .eslintignore:
-#   N/A
+.github

```

Comment:
Do we have to ignore .github completely?
Are the *.md files based on standard templates?
Does prettier corrupt the *.md files?

Reply:
Yes, yes and yes.

Reply:
The workflow-related *.yml files are most probably not going to be corrupted by prettier. We could include the workflow-related *.yml files ... 

```suggestion
.github
!.github/workflows/
```

... which doesn't work due to [lacking negation capabilities][issue] or exclude just the template directories (preferred):

```suggestion
.github/*_TEMPLATE/
```

[issue]: https://github.com/prettier/prettier/issues/4912